mappings/net/minecraft/client/realms/gui/screen/RealmsParentalConsentScreen.mapping

```diff
@@ -1,6 +1,7 @@
 CLASS net/minecraft/class_4400 net/minecraft/client/realms/gui/screen/RealmsParentalConsentScreen
 	FIELD field_22701 parent Lnet/minecraft/class_437;
 	FIELD field_26491 PRIVACY_INFO_TEXT Lnet/minecraft/class_2561;
+	FIELD field_26492 privacyInfoText Lnet/minecraft/class_5489;
```

Comment:
How does this one differ from the constant `Text` one?

Reply:
Constant is Text (static) and this is MultilineText (non-static). I followed the convention of capitalizing constants and camelcasing others, though that isn't documented.